### ISSUE TITLE
npm install fails 404 Not Found because of dependency's broken colors package

### ISSUE THREAD

**Issue by erikrenberg** [raineorshine/npm-check-updates#1032]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 12`

`npm install` currently doesn't work to install npm-check-updates:
`npm ERR! 404 Not Found - GET https://registry.npmjs.org/colors/-/colors-1.4.2.tgz - Not found`

The cause is that npm-check-updates has a dependency `cli-table` that currently tries to bring in a broken colors 1.4.2 package from NPM. This explains the issue: https://snyk.io/blog/open-source-npm-packages-colors-faker/

Here are the dependencies:
```
└─┬ npm-check-updates@12.1.0
  └─┬ cli-table@0.3.11
    └── colors@1.03
```
When/if cli-table is fixed that should fix this issue I guess. 

For now I know I can remove npm-check-updates temporarily from the dev-dependencies to work around it.

**Comment by raineorshine** [comment]
Thank you for documenting this! Hopefully cli-table merged the fix very soon. I will continue watching closely.

**Comment by raineorshine** [comment]
Looks like cli-table has been abandoned. I've switched to `cli-table3` which no longer depends on `colors`.

Published to `v16.7.13`
